datalad_registry/tasks.py

```diff
@@ -70,6 +97,37 @@ def _extract_annex_info(repo) -> InfoType:
     return info
 
 
+@celery.task
+def collect_dataset_uuid(urls: Optional[List[str]] = None) -> None:
```

Comment:
note: a list here is I guess to mimic the existing interface of `collect_dataset_info`  which takes a list of `ds_id, url` pairs... not sure yet if there is a benefit of "batching" multiple updates this way, and also makes name a bit misleading (should have been `collect_dataset_infos` or alike to signal that it is not a single one to be processed).

Reply:
Unbatched task.